### PR TITLE
[🧹 UNIFICATION CLEANUP] Separate out `admin.logs.v1` feature

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

- [Separate out admin.logs.v1 feature](https://github.com/wso2/identity-apps/pull/6773/commits/945bdcbc7dd21150b4fa40c2f452de7eb463f003)

### Related Issues
- https://github.com/wso2/product-is/issues/20802

### Related PRs

- N/A

### Checklist
- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: features/admin.logs.v1/api/useLogs.ts
Comment (pavinduLakshan) [inline features/admin.logs.v1/api/useLogs.ts:1]: shall we rename the file to `use-logs.ts` to preserve consistency?

File: features/admin.logs.v1/constants/index.ts
Comment (pavinduLakshan) [inline features/admin.logs.v1/constants/index.ts:1]: Shall we get rid of index files with this effort, if possible :)

File: features/admin.logs.v1/pages/audit-logs-page.tsx [hunk 1]
```diff
@@ -51,7 +52,7 @@ type AuditPagePropsInterface = IdentifiableComponentInterface;
  *
  * @returns Logs Page {@link React.ReactElement}
  */
-export const AuditLogsPage = (props: AuditPagePropsInterface) : ReactElement => {
+const AuditLogsPage = (props: AuditPagePropsInterface) : ReactElement => {
```
Comment (pavinduLakshan) [inline features/admin.logs.v1/pages/audit-logs-page.tsx:55]: Have we removed `export` because it is not being used?

File: features/index.ts [hunk 2]
```diff
@@ -38,6 +38,7 @@ export * from "./admin.extensions.v1/public-api";
 export * from "./admin.groups.v1/public-api";
 export * from "./admin.identity-providers.v1/public-api";
 export * from "./admin.identity-verification-providers.v1/public-api";
+export * from "./admin.logs.v1/public-api";
```
Comment (pavinduLakshan) [inline features/index.ts:41]: Shall we not add exports for new modules in the index file

**Review by pavinduLakshan — APPROVED** [review]
Suggested some optional improvements :)

File: features/admin.logs.v1/pages/audit-logs-page.tsx (hunk 1 above)
Comment (brionmario) [reply features/admin.logs.v1/pages/audit-logs-page.tsx:55]: Fixed with https://github.com/wso2/identity-apps/pull/6773/commits/b900c355f3611c11828b641120f7afe5ac85d382

File: features/admin.logs.v1/constants/index.ts
Comment (brionmario) [reply features/admin.logs.v1/constants/index.ts:1]: Fixed with https://github.com/wso2/identity-apps/pull/6773/commits/b900c355f3611c11828b641120f7afe5ac85d382

File: features/index.ts (hunk 2 above)
Comment (brionmario) [reply features/index.ts:41]: This will be required for the `rollup` build.

File: features/admin.logs.v1/api/useLogs.ts
Comment (brionmario) [reply features/admin.logs.v1/api/useLogs.ts:1]: Fixed with https://github.com/wso2/identity-apps/pull/6773/commits/b3d416ce1586c181a36123bc165e51d5bc196996

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**